keps/sig-network/NNNN-kubelet-probes/kep.yaml

```diff
@@ -0,0 +1,39 @@
+title: Pod Probes v2
+kep-number: NNNN
+authors:
+  - "@danwinship"
```

Comment:
(you should add yourself here)

Reply:
haha will do once I myself contribute something substantial here => I hope to as this KEP evolves since we still need to fill other sections, but as is in current state, let me refrain - that will also motivate me to keep working on my knowledge.